### ISSUE TITLE
The "done" sounds have a cat meowing in the background.

### ISSUE THREAD

**Issue by matthias-ccri** [johannesjo/super-productivity#2512]
The default "done" sound has a cat sound in the background, near the end. It's faint, but I can hear a clear "meow" every time it plays. It's not a huge issue but it's a little funny and I think it's worth mentioning. I'm also wondering who else has noticed it.

The file is https://github.com/johannesjo/super-productivity/blob/master/src/assets/snd/done2.mp3

As a user, the workaround is to switch the sound in the Settings, to something like "Sound 3"

### Your Environment

- Version used: 7.12.2
- Operating System and version: Ubuntu 20.04.5
- Desktop Environment: default
- Browser Name and version: N/A

### Can you reproduce this reliably?

Yes

### Console Output

N/A

### Error Log (Desktop only)

N/A

**Comment by Petemir** [comment]
Sorry, I am not hearing it :rofl: . I hear something like a flute but it's just reverb/leftover of the main sound... this is the _tail_ of the soundwave 

https://user-images.githubusercontent.com/287105/224947668-5e765edc-c711-4517-8baf-df77091913f2.mp4

**Comment by matthias-ccri** [comment]
I messed around in Audacity and I identified the noise in the spectrogram. I'm a novice but it looks like a different pitch than the main noise. Here it is highlighted (it's also visible at the bottom without the box)

![image](https://user-images.githubusercontent.com/18034092/225057992-51b6ca1a-b906-4586-8243-aedcd20000c1.png)

I was also able to remove the noise with a spectrogram editing tool. I can't attach mp3s to comments however. I'll see about making a PR
